### PR TITLE
Desktop version development

### PR DESCRIPTION
**Pull request by M0rrighan**
Portfolio project from Figma template - Milestone 4:  Desktop version
- Restructure .css codebase
- Use of media queries to make the desktop version 
- Use of display grid and flex to layout the elements

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,312 +1,401 @@
...
+                  required
+                ></textarea>
+              </li>
+              <li class="submit_btn">
+                <button type="submit" class="btn action_btn enabled">
+                  Get in Touch
+                </button>
+              </li>
+            </ul>
+          </form>
```
Comment (Shaher-11) [inline index.html:396]: Kindly Increase the width of the form as shown below the difference between the original design and the current project

#### Original design
right

### Current project
wrong1

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @M0rrighan,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The code is clean and nicely  structured:smiley::smiley:
We just need to make  one change to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: index.html (hunk 1 above)
Comment (M0rrighan) [reply index.html:396]: Hi @Shaher-11, thank you for taking the time to review my code. Inspecting my project page I noticed that the computed width is exactly the same size as the Figma template size. I attached a screenshot from the project preview and a screenshot from the template to compare.
Figma_template@1440
Portfolio_Form_preview@1440w

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi @M0rrighan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

## Optional improvements (Please, fix this) ☑️ 
- This is optional but I kindly invite you to do it, at the end of the footer you could see a gray part as the last background for form. I kindly invite you to do something about it.
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**